WooCommerce/Classes/ViewRelated/Products/Variations/ProductVariationsViewController.swift

```diff
@@ -148,14 +148,28 @@ final class ProductVariationsViewController: UIViewController {
 //
 private extension ProductVariationsViewController {
 
-    /// Set the title.
+    /// Set the title and navigation buttons.
     ///
     func configureNavigationBar() {
         removeNavigationBackBarButtonText()
         title = NSLocalizedString(
             "Variations",
             comment: "Title that appears on top of the Product Variation List screen."
         )
+        if product.variations.isNotEmpty && isAddProductVariationsEnabled {
```

Comment:
I wonder why we don't have a VM for this VC 🤔 
*Nothing to change here tho, just rambling*

Reply:
I'd be very interested in learning more about when we do or don't think a VM is needed!

Reply:
That could be a great and wide discussion, but lately, my quick take is:

> Add code to a VM, If there is any logic that you think it's worth unit test.

In this particular example, there is no "easy" way to test that this code is working as expected. But in a VM this would be fairly simple. 

```swift
func test_more_button_appears_when_the_preoduct_is_not_empty_anf_feature_is_enabled() {
   let product = createNonEmptyProduct()
   let VM = ViewModel(product: Product, isFeatureEnabled: true)
   XCTAssertEquals(vm.showMoreButton, true)
}
```

And the same for all the other combinations!